other/bootstrap_daemon/tox-bootstrapd-tmpfile.conf

```diff
@@ -0,0 +1 @@
+d /var/run/tox-bootstrapd 0750 tox-bootstrapd tox-bootstrapd -
```

Comment:
I see [it's suggested](http://www.freedesktop.org/software/systemd/man/tmpfiles.d.html) to use `RuntimeDirectory=` (and `RuntimeDirectoryMode=`) instead of tempfile for PID files, sockets, etc.

>System daemons frequently require private runtime directories below /run to place communication sockets and similar in. For these, consider declaring them in their unit files using RuntimeDirectory= (see systemd.exec(5) for details), if this is feasible.